documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc

```diff
@@ -1,20 +1,7 @@
 [id='generating_protostream_sci-{context}']
-= Generating Serialization Context Initializers
```

Comment:
I still think that the section on generation `SerializationContextInitializers` is relevant as you still need todo this if you're using annotated classes or an adaptor class to generate your proto schema and marshaller. Imo we should keep this section and then in the "Marshalling Custom Java Objects" and "Marshalling External Classes" sections, we simply reference this when talking about class registration with SerializationContextInitializer as the process is the same for both.

A valid use-case, which I think we should mention in the docs, is to have a single `SerializationContextInitializer` definition that contains both annotated and adapter classes.